packages/core/src/connectivity/scp-cf/destination/destination-cache.spec.ts

```diff
@@ -140,6 +142,24 @@ describe('caching destination integration tests', () => {
       );
     });
 
+    it('cache key contains user also for provider tokens', async () => {
```

Comment:
This is the test which failed without the test. It would contain a key: `tenant:undefined:destName`. With your fix merged it would contain no cache key at all I guess.